src/main/java/bytetalker/ui/Ui.java

```diff
@@ -0,0 +1,144 @@
+package bytetalker.ui;
+
+import bytetalker.task.Task;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Deals with interactions with the user. Mainly the print statements are included in this class to control the
+ * messages.
+ *
+ * @author Junseo Kim
+ * @version 0.1
+ * @since 2024-02-06
+ */
+public class Ui {
+    /**
+     * Prints out welcome message and command format the user can use to enter the task.
+     */
+    public void showWelcome() {
+        System.out.println("    -----------------------------------");
+        System.out.println("    Hello! I'm bytetalker.ByteTalker");
+        System.out.println("    What can I do for you?");
+        System.out.println("    Supported tasks are todo, deadline, and event");
+        System.out.println("    todo {task}");
+        System.out.println("    deadline {task} /by {date and time}");
+        System.out.println("    event {task} /from {date and time} /to {date and time}");
+        System.out.println("    Supported date and time format are yyyy-MM-dd HHmm, and dd/MM/yyyy HHmm");
+        System.out.println("    -----------------------------------");
+    }
+
+    /**
+     * Prints out bye message.
+     */
+    public void showBye() {
+        System.out.println("    -----------------------------------");
+        System.out.println("    Bye. Hope to see you again soon!");
+        System.out.println("    -----------------------------------");
+    }
+
+    /**
+     * Prints out error message if there is an error while changing the task list.
+     */
+    public void showErrorMsgStoreTasks() {
```

Comment:
Maybe an alternate name to consider would be "showStoreTaskErrorMessage"

Reply:
That seems to make more sense. Thank you for feedback!